### ISSUE TITLE
Error token_cache

### ISSUE THREAD

**Issue by Flavien06** [alfem/synology-flickr-folder-uploader#1]
Hi
The script doesn't show me URL to authorize upload.
I have this error:

> Uploading /volume1/photo/Pictures/ to Pictures with tags: synology-nas-uploaded
Traceback (most recent call last):
  File "./flickr-folder-uploader.py", line 44, in <module>
    flickr = flickrapi.FlickrAPI(API_KEY, API_SECRET, token_cache_location=TOKEN_CACHE)
NameError: name 'TOKEN_CACHE' is not defined

**Comment by hohodyret** [comment]
I have the same problem @Flavien06 , did you solve the issue?

**Comment by Flavien06** [comment]
Yes, juste solve it
I have missed it, in the script: delete the "#"
#TOKEN_CACHE='. /token'.
in
TOKEN_CACHE='. /token'.


But now I have

> Authentication required
> https://www.flickr.com/services/oauth/authorize?oauth_token=xxxxxxxxxxxxxxxxxxxxxxxxx-xxxxxxxxxxxxxxxx&perms=write
> Verifier code: 104-902-660
> Verifier: 104-902-660
> None
> Creating set Pictures ERROR.

**Comment by oPromessa** [comment]
@Flavien06 in case it might help you/interest you take a look at this fork [flickr-uploader](https://github.com/oPromessa/flickr-uploader) and also available [on pypi.org also](https://pypi.org/project/flickr-uploader/).

**Comment by alfem** [comment]
Well I finally bought a new QNAP nas and sold my Synology, so I can not make further testing :-)

**Comment by alfem** [comment]
> @Flavien06 in case it might help you/interest you take a look at this fork [flickr-uploader](https://github.com/oPromessa/flickr-uploader) and also available [on pypi.org also](https://pypi.org/project/flickr-uploader/).

It is not a fork of this code, is it?

**Comment by oPromessa** [comment]
No, it's a for from another project trickortwak/flickr-uploader which references your project.

**Comment by alfem** [comment]
Could you point at the page referencing my project? I can not find anything like that in trickortwak/flickr-uploader.

**Comment by oPromessa** [comment]
Sorry.. the project pages are:
* [trickortweak/flickr-uploader](https://github.com/trickortweak/flickr-uploader)
* The updated fork is [oPromessa/flickr-uploader](https://github.com/oPromessa/flickr-uploader)
   * It should also run on QNAP. I would be curious to know. If you try it, let me know!

**Comment by alfem** [comment]
But none of those projects references this.

**Comment by oPromessa** [comment]
Yup it does! ;) [Your comment from June 2017 on this link](https://github.com/trickortweak/flickr-uploader/issues/86#issuecomment-309697894)

**Comment by alfem** [comment]
So you are just hunting users around, aren't you?  ;-)

**Comment by oPromessa** [comment]
Ah ah ah... well... hmmm... trying to help out.... but... hmmm. Yes, you nailed it!
